search-books/src/main/java/nix/education/request/BookRequest.java

```diff
@@ -0,0 +1,22 @@
+package nix.education.request;
+
+import java.util.Date;
+import java.util.List;
+import lombok.Data;
+import lombok.SneakyThrows;
+import nix.education.util.DateFormat;
+
+@Data
+public class BookRequest {
+
+    private String author;
+    private String title;
+    private String description;
+    private List<String> tags;
+    private String publication_date;
+
+    @SneakyThrows
+    public Date getPublication_date() {
```

Comment:
Java naming convention discourage usage of underscore sign in class/method/field names

Reply:
Refactored